addons/l10n_mt/data/template/account.fiscal.position-mt.csv

```diff
@@ -0,0 +1,33 @@
+"id","name","sequence","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","name@hr"
```

Comment:
mistake here should be mt, but since the language of malta is for 88% of the population English i think we can remove the column